Tools/autotest/apmrover2.py

```diff
@@ -47,6 +53,9 @@ def is_rover(self):
     def get_stick_arming_channel(self):
         return int(self.get_parameter("RCMAP_ROLL"))
 
+    def load_arming_test_mission(self):
```

Comment:
Maybe a method which returns the path instead?

Should possibly make it more clear what the arming test mission is supposed to accomplish.

Reply:
yep.
The arming test mission does nothing, it got a unlimited wait . It is need for plane and rover otherwise the vehicle refuse to change to auto without mission

Reply:
done